src/components/ScholarshipForm.jsx

```diff
@@ -229,42 +247,58 @@ function ScholarshipForm({ scholarship, submitFn, onSubmitError }) {
     </Grid>
   );
 
+  function validationCheck() {
+    // no requirements and checkbox not checked
+    if (
+      !Object.values(formik.values.requirements).some((val) => {
+        if (typeof val == 'number') return val != 0;
+        return val.length != 0;
+      }) &&
+      !noReqsChecked
+    ) {
+      setNoReqsHelperText(
+        'Check this box if there are no requirements for this scholarship.'
+      );
+      return;
+    }
+    setNoReqsHelperText('');
+    formik.submitForm();
```

Comment:
This is missing the (unexpected) case where no requirements is checked but there are actually requirement values.

If you default `requirements` to `{}` for new scholarships this can be written as:
```js
      const noReqsGiven = Object.keys(formik.values.requirements).length !== 0;
      if (noReqsGiven === noReqsChecked) {
        formik.submitForm();
      } else if (noReqsGiven) {
        setNoReqsHelperText('Check this box if there are no requirements for this scholarship.');
      } else {
        setNoReqsHelperText('Uncheck this box if there are requirements for this scholarship.');
      }
```